packages/dapp/tsconfig.json

```diff
@@ -26,6 +26,6 @@
       "@/types/contracts": ["types/contracts"]
     }
   },
-  "include": ["global.d.ts", "next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["global.d.ts", "next-env.d.ts", "**/*.ts", "**/*.tsx", "ubq.config.ts"],
```

Comment:
The include should be updated to `ubq-connect-config.ts`

Reply:
pleased at https://github.com/ubiquity/ubiquity-dollar/pull/888/commits/df1e6deda304c2ed4494e030e2cfb2748a791848